awscli/customizations/s3/filegenerator.py

```diff
@@ -324,4 +326,4 @@ def _list_single_object(self, s3_path):
         file_size = int(response['ContentLength'])
         last_update = parse(response['LastModified'])
         last_update = last_update.astimezone(tzlocal())
-        return s3_path, file_size, last_update
```

Comment:
This seems like a weird API.  We're returning parts of the response (such as LastModified) as well as just the entire response whole sale.  This can probably be cleaned up.